src/components/play/fillInTheBlanks/Answers.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { Box } from '@mui/material';
+
+import Answer from './Answer';
+
+const Answers = ({ answers }) => {
+  const onDragStart = (e, id) => {
+    if (e.dataTransfer) {
+      e.dataTransfer.setData('text/plain', id);
+    }
+  };
+
+  const _handleDragOver = (e) => {
+    e.preventDefault();
+  };
+
+  const _handleDragLeave = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <Box
+      display="flex"
+      p={1}
+      justifyContent="center"
+      mb={3}
+      sx={{
+        borderRadius: '10px',
+        border: '1px grey dotted',
+        flexWrap: 'wrap',
+        minHeight: '50px',
+        width: '100%',
```

Comment:
These keys can be set directly as props on the Box component (it supports all props from the system [see Box API](https://mui.com/material-ui/api/box/) and [system props](https://mui.com/system/properties/)

Reply:
Incredible!